packages/amplify-provider-awscloudformation/src/graphql-transformer/amplify-graphql-resource-manager.ts

```diff
@@ -142,31 +139,35 @@ export class GraphQLResourceManager {
       const tableNames = [];
       tables.forEach(tableName => {
         tableNames.push(tableNameMap.get(tableName));
-        const filepath = path.join(stateFileDir, stepNumber, 'stacks', `${tableName}.json`);
+        const filepath = path.join(stepPath, 'stacks', `${tableName}.json`);
         fs.ensureDirSync(path.dirname(filepath));
         JSONUtilities.writeJson(filepath, this.templateState.pop(tableName));
       });
 
       const deploymentRootKey = `${ROOT_APPSYNC_S3_KEY}/${buildHash}/states/${stepNumber}`;
       const deploymentStep: DeploymentOp = {
```

Comment:
`deploymentStepState` ?

Reply:
This filepath would more specifically be the `tableStackFilePath`